openocd.gdb

```diff
@@ -10,30 +10,25 @@ set backtrace limit 32
 break DefaultHandler
 break HardFault
 break rust_begin_unwind
-# # run the next few lines so the panic message is printed immediately
-# # the number needs to be adjusted for your panic handler
-# commands $bpnum
-# next 4
-# end
 
 # *try* to stop at the user entry point (it might be gone due to inlining)
 break main
 
-set mem inaccessible-by-default off
+monitor arm semihosting enable
```

Comment:
Why enable the semihosting crutch if you want to use ITM?

Reply:
Oops, you're right. That's got to go.